### ISSUE TITLE
Update Quantos S.O.P.

### ISSUE THREAD

**Issue by sonyahanson** [choderalab/wetlab-protocols#41]
Note there is already this issue: https://github.com/choderalab/lab-protocols/issues/35

The main thing from my perspective that needs to be updated is:
 - adjust label printing so the name is not cut off

This email from @MehtapIsik is perhaps relevant, maybe @jchodera would want this zip file:
![quantos_label](https://cloud.githubusercontent.com/assets/6333823/25346186/eec11f4e-28e4-11e7-9e29-3463da60b62c.png)

In a previous wetlab meeting we decided these were the fields we wanted on the labels in order of priority (ideal to have them all on the label, but if they won't fit we want at least primary and secondary details):

Primary:
 - Substance name
 - Fill date
 - True concentration
 - UUID (has to be entered by user manually)

Secondary:
 - Supplier
 - LOT #
 - CAT #

Less important:
 - username
 - expiration date

I think @steven-albanese also mentioned:
-  not using 'safe position' because it is in fact, not very safe.

Anything else?

**Comment by MehtapIsik** [comment]
In my opinion optimizing **validation and release settings** of the following methods have higher priority:
- GSP stable
- liquid
- powder

I observe that the taring step before dispense is not ideal. It zeros at the beginning but the value keeps increasing before dispensing starts.

**Comment by sonyahanson** [comment]
I have never experienced this, but I agree it's a higher priority. Has this only been happening since January?

**Comment by MehtapIsik** [comment]
Yes, after they changed the balance. Our previous stabilization settings are not fitting anymore.

**Comment by jchodera** [comment]
We've so far determined that there is an issue with the antistatic kit never activating when operating the Quantos through LabX. We've asked Emily Stawicki of Mettler-Toledo for guidance, since the manual has nothing useful to say about how to inform LabX that the Quantos has antistatic capability.

In the meantime, we have created a protocol `GSP stable (04/26/2017)" that makes the following "best guesses":
* Weighing Mode: Universal
* Environment: Very Unstable 
* Value Release: Very Reliable
* AutoZero: Yes

This protocol *may* still suffer from both issues from static electricity and issues with small upward drift after zeroing that are masked by the long delay before reporting caused by "Value Release: Very Reliable".

Note that it is possible to disconnect the Quantos from LabX and go through solid/liquid/solution dosing directly from the balance using a completely uncoupled set of protocols. It could be that we want to get these Quantos-only protocols working to our standards on their own before porting them back over to LabX. The antistatic kit works fine if operating the Quantos disconnected from LabX.

We haven't yet addressed just solid or liquid dosing methods while we wait for clarification from Mettler.
